Add LVM example to Filebeat file / device ID docs

kilfoyle opened the pull request:
This updates the "[file identity](https://www.elastic.co/guide/en/beats/filebeat/8.12/filebeat-input-filestream.html#filestream-file-identity)" section on the Filebeat `filestream` input page to give an explicit example of where `filestream` should be set to `path`. 

Closes: #34418

![screen](https://github.com/elastic/beats/assets/41695641/964f5726-a12e-49bb-8c0c-f3626d5b20f7)

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-metricbeat/builds/2412)
* Commit: bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-metricbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-metricbeat/builds/2412","number":2412,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":2412}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-libbeat/builds/1199)
* Commit: bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-libbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-libbeat/builds/1199","number":1199,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1199}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-packetbeat/builds/1194)
* Commit: bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-packetbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-packetbeat/builds/1194","number":1194,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1194}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-winlogbeat/builds/363)
* Commit: bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-winlogbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-winlogbeat/builds/363","number":363,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":363}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-37959/detail/PR-37959/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-37959/detail/PR-37959/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-37959/detail/PR-37959/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-37959/detail/PR-37959/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://beats_bk_37959.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/beats-ci/transactions/view?rangeFrom=2024-02-09T15:06:48.723Z&rangeTo=2024-02-09T15:26:48.723Z&transactionName=BUILD+Beats%2Fbeats%2FPR-%7Bnumber%7D&transactionType=job&latencyAggregationType=avg&traceId=6c277394c28cb5d78d4b560321d832df&transactionId=4b530cb93bb29908)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats





* Duration: 9 min 2 sec





</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->














## :grey_exclamation: Flaky test report
No test was executed to be analysed.



## :robot: GitHub comments

<details><summary>Expand to view the GitHub comments</summary>
<p>

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.


- `/package` : Generate the packages and run the E2E tests.


- `/beats-tester` : Run the installation tests with beats-tester.


- `run` `elasticsearch-ci/docs` : Re-trigger the docs validation. (use unformatted text in the comment!)




</p>
</details>

<!--COMMENT_GENERATED_WITH_ID_comment.id-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/filebeat/builds/2865)
* Commit: bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

### Failed CI Steps
* [:windows:-family/core-windows-2016 Unit Tests](https://buildkite.com/elastic/filebeat/builds/2865#018d8e72-9819-4d93-847f-e39068294b80)

cc @kilfoyle

<!--buildkite-pr-comment-filebeat
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/filebeat/builds/2865","number":2865,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":2865}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/heartbeat/builds/1526)
* Commit: bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-heartbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/heartbeat/builds/1526","number":1526,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1526}
buildkite-pr-comment-->

kilfoyle commented:
@belimawr Yeah, that makes perfect sense. Thanks! I've moved the new text to under the `native` option, in the file_identity section. I've updated the screen capture in the PR description.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-packetbeat/builds/1261)
* Commit: c0142bc0aefffc6e27a4b90f719ce5118e74e5ba

### History
* :green_heart: [Build #1194](https://buildkite.com/elastic/beats-packetbeat/builds/1194) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-packetbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-packetbeat/builds/1261","number":1261,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-packetbeat/builds/1194","number":1194,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1261}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-libbeat/builds/1267)
* Commit: c0142bc0aefffc6e27a4b90f719ce5118e74e5ba

### History
* :green_heart: [Build #1199](https://buildkite.com/elastic/beats-libbeat/builds/1199) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-libbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-libbeat/builds/1267","number":1267,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-libbeat/builds/1199","number":1199,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1267}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-metricbeat/builds/2481)
* Commit: c0142bc0aefffc6e27a4b90f719ce5118e74e5ba

### History
* :green_heart: [Build #2412](https://buildkite.com/elastic/beats-metricbeat/builds/2412) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-metricbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-metricbeat/builds/2481","number":2481,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-metricbeat/builds/2412","number":2412,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":2481}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-winlogbeat/builds/432)
* Commit: c0142bc0aefffc6e27a4b90f719ce5118e74e5ba

### History
* :green_heart: [Build #363](https://buildkite.com/elastic/beats-winlogbeat/builds/363) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-winlogbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-winlogbeat/builds/432","number":432,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-winlogbeat/builds/363","number":363,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":432}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/heartbeat/builds/1592)
* Commit: c0142bc0aefffc6e27a4b90f719ce5118e74e5ba

### History
* :green_heart: [Build #1526](https://buildkite.com/elastic/heartbeat/builds/1526) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-heartbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/heartbeat/builds/1592","number":1592,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/heartbeat/builds/1526","number":1526,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1592}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/filebeat/builds/2931)
* Commit: c0142bc0aefffc6e27a4b90f719ce5118e74e5ba

### History
* :broken_heart: [Build #2865](https://buildkite.com/elastic/filebeat/builds/2865) failed bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-filebeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/filebeat/builds/2931","number":2931,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/filebeat/builds/2865","number":2865,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":2931}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-libbeat/builds/1498)
* Commit: db45c6a9705cb46a4d1212ed2d6129001e2aafd5

### History
* :green_heart: [Build #1267](https://buildkite.com/elastic/beats-libbeat/builds/1267) succeeded c0142bc0aefffc6e27a4b90f719ce5118e74e5ba
* :green_heart: [Build #1199](https://buildkite.com/elastic/beats-libbeat/builds/1199) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-libbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-libbeat/builds/1498","number":1498,"commit":"db45c6a9705cb46a4d1212ed2d6129001e2aafd5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-libbeat/builds/1267","number":1267,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-libbeat/builds/1199","number":1199,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1498}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-metricbeat/builds/2712)
* Commit: db45c6a9705cb46a4d1212ed2d6129001e2aafd5

### History
* :green_heart: [Build #2481](https://buildkite.com/elastic/beats-metricbeat/builds/2481) succeeded c0142bc0aefffc6e27a4b90f719ce5118e74e5ba
* :green_heart: [Build #2412](https://buildkite.com/elastic/beats-metricbeat/builds/2412) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-metricbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-metricbeat/builds/2712","number":2712,"commit":"db45c6a9705cb46a4d1212ed2d6129001e2aafd5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-metricbeat/builds/2481","number":2481,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-metricbeat/builds/2412","number":2412,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":2712}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-packetbeat/builds/1506)
* Commit: db45c6a9705cb46a4d1212ed2d6129001e2aafd5

### History
* :green_heart: [Build #1261](https://buildkite.com/elastic/beats-packetbeat/builds/1261) succeeded c0142bc0aefffc6e27a4b90f719ce5118e74e5ba
* :green_heart: [Build #1194](https://buildkite.com/elastic/beats-packetbeat/builds/1194) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-packetbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-packetbeat/builds/1506","number":1506,"commit":"db45c6a9705cb46a4d1212ed2d6129001e2aafd5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-packetbeat/builds/1261","number":1261,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-packetbeat/builds/1194","number":1194,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1506}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/beats-winlogbeat/builds/662)
* Commit: db45c6a9705cb46a4d1212ed2d6129001e2aafd5

### History
* :green_heart: [Build #432](https://buildkite.com/elastic/beats-winlogbeat/builds/432) succeeded c0142bc0aefffc6e27a4b90f719ce5118e74e5ba
* :green_heart: [Build #363](https://buildkite.com/elastic/beats-winlogbeat/builds/363) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-beats-winlogbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-winlogbeat/builds/662","number":662,"commit":"db45c6a9705cb46a4d1212ed2d6129001e2aafd5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-winlogbeat/builds/432","number":432,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/beats-winlogbeat/builds/363","number":363,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":662}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/heartbeat/builds/1800)
* Commit: db45c6a9705cb46a4d1212ed2d6129001e2aafd5

### History
* :green_heart: [Build #1592](https://buildkite.com/elastic/heartbeat/builds/1592) succeeded c0142bc0aefffc6e27a4b90f719ce5118e74e5ba
* :green_heart: [Build #1526](https://buildkite.com/elastic/heartbeat/builds/1526) succeeded bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-heartbeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/heartbeat/builds/1800","number":1800,"commit":"db45c6a9705cb46a4d1212ed2d6129001e2aafd5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/heartbeat/builds/1592","number":1592,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/heartbeat/builds/1526","number":1526,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":1800}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/filebeat/builds/3139)
* Commit: db45c6a9705cb46a4d1212ed2d6129001e2aafd5

### History
* :green_heart: [Build #2931](https://buildkite.com/elastic/filebeat/builds/2931) succeeded c0142bc0aefffc6e27a4b90f719ce5118e74e5ba
* :broken_heart: [Build #2865](https://buildkite.com/elastic/filebeat/builds/2865) failed bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55

cc @kilfoyle

<!--buildkite-pr-comment-filebeat
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/filebeat/builds/3139","number":3139,"commit":"db45c6a9705cb46a4d1212ed2d6129001e2aafd5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/filebeat/builds/2931","number":2931,"commit":"c0142bc0aefffc6e27a4b90f719ce5118e74e5ba"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/filebeat/builds/2865","number":2865,"commit":"bbdc09ef1f85d6aa3a1e677f6e9f027e05029c55"}],"number":3139}
buildkite-pr-comment-->
